packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -287,6 +303,8 @@ class Wizard extends React.Component<WizardProps> {
     const computedSteps: ComputedStep[] = this.initSteps(steps, activeStep);
     const firstStep = activeStep === flattenedSteps[0];
     const isValid = activeStep.enableNext !== undefined ? activeStep.enableNext : true;
+    const setFullWidth = isFullWidth || !!(width)
```

Comment:
Shouldn't this be we want to set these booleans if the width/height exist :
```suggestion
    const setFullWidth = isFullWidth || width
    const setFullHeight = isFullHeight || height
```